warp-tls/Network/Wai/Handler/WarpTLS.hs

```diff
@@ -129,6 +131,15 @@ data TLSSettings = TLSSettings {
     -- Default: Nothing
     --
     -- Since 3.2.2
+  , tlsSessionManagerConfig :: Maybe SM.Config
+    -- ^ Configuration for in-memory TLS session manager.
+    -- If Nothing, 'TLS.noSessionManager' is used.
```

Comment:
- It changes the current behavior of warp-tls (i.e. allowing session resumptions).
- It requires more memory than the current.

So, I think that users should explicitly set this parameter.